tests/bluetooth/bsim/mesh/src/test_dfu.c

```diff
@@ -281,6 +371,8 @@ static const struct bt_mesh_comp target_comp = {
 						BT_MESH_MODEL_CFG_CLI(&cfg_cli),
 						BT_MESH_MODEL_SAR_CFG_SRV,
 						BT_MESH_MODEL_SAR_CFG_CLI(&sar_cfg_cli),
+						BT_MESH_MODEL_CB(IMPOSTER_MODEL_ID,
```

Comment:
This is needed because to fail after start we need imposter model that hijacks caps get message (like one target in fail_on_persistency). But after this I want to reboot device and setup it with composition that handles all messages properly - with normal `target_comp`. This caused problem, because there was different model count: when model settings were restored, there was no model with this index, which caused error.

Reply:
Could you please add a comment about this?

Reply:
sure :)